.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "jest.pathToJest": "npm run jest --",
+  "jest.pathToJest": "yarn jest --",
```

Comment:
Works nicely, but it should be yarn to fit with the rest of the dev recommendations

![screen shot 2017-05-19 at 16 02 17](https://cloud.githubusercontent.com/assets/49038/26253823/8d0e5e70-3cac-11e7-9f42-1d7e2df3c802.png)